### PR TITLE
fix endpoint domain option

### PR DESCRIPTION
**Pull request by tscanlin**
@mauerbac & @nchilada 

This fixes the endpoint_domain option. Good catch!

### CODE REVIEW

File: src/partials/_sidebyside.html [hunk 1]
```diff
@@ -123,7 +123,7 @@ <h3 id="{{object.anchor}}">
   -H "Token: abcdefghijklmnopqrstuvwxyz:123456" \{% if object.request %}
   -H "Content-Type: application/json" \
   -d '{{ object.request | dump | safe }}' \{% endif %}
-  {% if type %}-X {{object.type}} {% endif %}"{% if endpoint_domain %}{{ endpoint_domain }}{% else %}https://www.optimizelyapis.com/experiment/v1/{% endif %}{{ object.endpoint | safe }}"
```
Comment (nchilada) [inline src/partials/_sidebyside.html]: It's really weird that Nunjucks is able to find a default value when it sees `endpoint_domain` without the `object.` namespace.  Maybe we define `endpoint_domain` in two different scopes.

**Comment by nchilada** [comment]
LGTM.  Thanks for the fix!

File: src/partials/_sidebyside.html (hunk 1 above)
Comment (tscanlin) [inline src/partials/_sidebyside.html]: I think nunjucks isn't able to see endpoint_domain which is why it defaults to the normal endpoint. Seems like expected behavior to me.

**Comment by tscanlin** [comment]
NP, thank you!

File: src/partials/_sidebyside.html (hunk 1 above)
Comment (nchilada) [inline src/partials/_sidebyside.html]: I see, thanks.

**Comment by mauerbac** [comment]
Awesome, thanks @tscanlin!!

**Pull request merged**